Backpack/Calendar/Classes/BPKCalendar.m

```diff
@@ -1,5 +1,103 @@
 #import "BPKCalendar.h"
+#import "BPKCalendarHeaderView.h"
+
+#import <Backpack/Color.h>
+#import <Backpack/Font.h>
+#import <FSCalendar/FSCalendar.h>
+
+@interface BPKCalendar () <FSCalendarDelegate, FSCalendarDelegateAppearance, FSCalendarDataSource>
+
+@property (nonatomic) FSCalendar *calendarView;
+
+@end
 
 @implementation BPKCalendar
 
-@end
\ No newline at end of file
+- (instancetype)initWithCoder:(NSCoder *)coder
+{
+    self = [super initWithCoder:coder];
+    if (self) {
+        [self setupViews];
+        [self addViews];
+        [self setupConstraints];
+    }
+    return self;
+}
+
+- (instancetype)initWithFrame:(CGRect)frame
+{
+    self = [super initWithFrame:frame];
+    if (self) {
+        [self setupViews];
+        [self addViews];
+        [self setupConstraints];
+    }
+    return self;
+}
+
+- (void)setupViews {
+    self.translatesAutoresizingMaskIntoConstraints = NO;
+    self.calendarView = [[FSCalendar alloc] initWithFrame:CGRectZero];
+    self.calendarView.translatesAutoresizingMaskIntoConstraints = NO;
+    self.calendarView.scrollDirection = FSCalendarScrollDirectionVertical;
+    self.calendarView.scrollEnabled = YES;
+    self.calendarView.pagingEnabled = NO;
+    self.calendarView.placeholderType = FSCalendarPlaceholderTypeNone;
+    self.calendarView.delegate = self;
+    self.calendarView.dataSource = self;
+    
+    NSDictionary<NSAttributedStringKey, id> *monthTextAttributes = [BPKFont attributesForFontStyle:BPKFontStyleTextLgEmphasized];
+    NSDictionary<NSAttributedStringKey, id> *weekdayTextAttributes = [BPKFont attributesForFontStyle:BPKFontStyleTextSm];
+    
+    FSCalendarAppearance *appearance = self.calendarView.appearance;
+    appearance.headerDateFormat = @"MMMM";
+    appearance.headerTitleFont = monthTextAttributes[NSFontAttributeName];
```

Comment:
As previously discussed this will not ensure correct usage of the type scale. Can we render the header ourselves instead of using this method?

Reply:
I found a way in the meantime for this. Fixing it right now

Reply:
Fixing this involved some hacks see the part where I reassign the appearance instance